src/starknet_cli_wrapper.py

```diff
@@ -30,13 +31,26 @@ async def starknet_compile_main_wrapper(args):
         print(err, file=sys.stderr)
         return 1
 
+async def cargo_run(args):
```

Comment:
Why is this a part of the starknet_cli_wrapper? The purpose of this internal server is to have things pre-imported so that they don't have to be imported on each starknet-related call. And cargo has nothing to do with Python imports.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/328/commits/3c8a4677f1890a99bb80af399f3409ac5f20280c